src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String csvHeader = "fruit,quantity";
+    private static final String elementSeparator = ",";
+    private static final String lineSeparator = System.lineSeparator();
```

Comment:
```suggestion
    private static final String CSV_HEADER = "fruit,quantity";
    private static final String elementSeparator = ",";
    private static final String lineSeparator = System.lineSeparator();
```
the same for other fields